tests/nnstreamer_filter_mvncsdk2/unittest_filter_mvncsdk2.cc

```diff
@@ -0,0 +1,117 @@
+/**
+ * @file    unittest_filter_mvncsdk2.cc
+ * @date    10 Jan 2020
+ * @brief   Unit test for the tensor filter sub-plugin for MVNCSDK2
+ * @see     https://github.com/nnsuite/nnstreamer
+ * @author  Wook Song <wook16.song@samsung.com>
+ * @bug     No known bugs.
+ */
+
+#include <string.h>
+#include <gtest/gtest.h>
+#include <gst/gst.h>
+#include <gst/check/gstcheck.h>
+#include <gst/check/gsttestclock.h>
+#include <gst/check/gstharness.h>
+#include <glib/gstdio.h>
+#include <tensor_common.h>
+#include <nnstreamer_plugin_api_filter.h>
+
+#include "NCSDKTensorFilterTestHelper.hh"
+
+/**
+ * @brief Negative case with incorrect tensor meta
+ */
+TEST (pipeline_mvncsdk2_filter, launch_normal)
+{
+  const gchar *root_path = g_getenv ("NNSTREAMER_BUILD_ROOT_PATH");
+  gchar *pipeline;
+  gchar *test_model;
+  GstElement *gstpipe;
+  GError *err = NULL;
+  NCSDKTensorFilterTestHelper *instance;
+  int status = 0;
+
+  if (root_path == NULL) {
+    root_path = "..";
+  }
+  test_model = g_build_filename (root_path, "tests", "test_models", "models",
+      "google_lenet_ncsdk_caffe_1.graph", NULL);
+  /* Create a nnstreamer pipeline */
+  pipeline = g_strdup_printf ("videotestsrc ! videoconvert ! videoscale ! videorate ! video/x-raw,format=BGR,width=224,height=224 "
+      "! tensor_converter ! tensor_transform mode=arithmetic option=typecast:float32,add:-104.0069877 "
+      "! tensor_filter framework=movidius-ncsdk2 model=\"%s\" ! fakesink",
+      test_model);
+
+  instance = &(NCSDKTensorFilterTestHelper::getInstance());
+  instance->init (GOOGLE_LENET);
```

Comment:
Q: when `.release()` is called?

Reply:
In this case, .release () is unecessary.